core/templates/pages/exploration-editor-page/suggestion-modal-for-editor-view/exploration-editor-suggestion-modal.controller.ts

```diff
@@ -38,9 +38,7 @@ angular.module('oppia').controller(
       $scope.canAccept = $scope.canEdit && $scope.isNotHandled &&
         suggestionIsValid && !unsavedChangesExist;
 
-      if (!$scope.canEdit) {
-        $scope.errorMessage = '';
```

Comment:
Why this change?

Reply:
Because it follows the same logic of the `else` block on line 53, which means I could just evaluate expressions where `$scope.errorMessage` could be different from '' and all the remaining expressions should be evaluated on `else` block.